jmbitcoin/jmbitcoin/secp256k1_transaction.py

```diff
@@ -81,7 +106,7 @@ def human_readable_output(txoutput):
         pass # non standard script
     return outdict
 
-def estimate_tx_size(ins, outs, txtype='p2pkh'):
+def estimate_tx_size(ins, outs, txtype='p2pkh', outtype=None):
```

Comment:
The docstring for the function should have the section "we assume the output types are the same as the input" edited, since that's the point of this new kwarg.

Reply:
Addressed in https://github.com/JoinMarket-Org/joinmarket-clientserver/pull/921/commits/635f3f10bc2628dbed8a213527659dc9f6b48616